src/content/1.7/scale/webserver.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Optimize your webserver
+alias:
+- 1.7/optimizations/
+---
+
+# Optimize your webserver
+
+We often talk about optimizing the application side but less about the webserver.
+
+Which is a shame as there is a lot to gain from some simple configurations parameters. And you don't need to have hundred of visitors for them to be efficient, either to reduce your server's load or speed up your visitor's browsing.
+
+As we'll see, the configuration options we'll tweak are quite easy and should allow some nice improvements.
```

Comment:
On devdocs we try to have a neutral and "official" tone so we avoid shortcuts 😉
```suggestion
As we will see, the configuration options we'll tweak are quite easy and should allow some nice improvements.
```

Same for "don't need" => "do not need", and this kind of things 😄 we try to look professionnal

Reply:
Tried to rework that in new version, let me know what you think ;)